build-config/make/images.make

```diff
@@ -378,7 +405,11 @@ $(RECOVERY_INITRD_STAMP): $(IMAGE_UPDATER_STAMP)
 	$(Q) touch $@
 
 ifeq ($(UEFI_ENABLE),yes)
-  RECOVERY_XORRISO_OPTIONS = $(RECOVERY_CONF_DIR)/xorriso-options-uefi.cfg
+  ifeq ($(ARCH),arm64)
```

Comment:
Here I would not check for $(ARCH).
I would rename the original `xorriso-options-uefi.cfg` to now contain the arch, something like:
```
xorriso-options-x86_64.cfg
```
Then this file can simply refer to `xorriso-options-$(ARCH)-uefi.cfg`.